libs/components/src/lib/Typography/index.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import Typography, { TypographyProps } from "./index";
+
+const identity = "typography text";
```

Comment:
use uppercase for constants and use @faker-js/faker for fake values like this:
const IDENTITY = faker.word.noun();